gocd/values.yaml

```diff
@@ -370,8 +376,21 @@ agent:
     #     secretName: github-key
     #     defaultMode: 0744
 
+    # agent.persistence.claimTemplates additional volumes dedicated to each replica
+    claimTemplates: []
```

Comment:
Maybe should just name as `volumeClaimTemplates` the same as in the pod spec if we intend to keep it?

Reply:
Yeah, not a fan of overly long names, but I WAS conflicted.